SAPHana/bin/SAPHanaSR-manageAttr

```diff
@@ -670,31 +680,110 @@ chk_multi_target_requirements() {
     return $chk
 }
 
+# get resource name from cib
+get_res() {
+    local resType=$1
+    cib=/var/lib/pacemaker/cib/cib.xml
+    # get resources for given $RSID
+    resS="$(grep '<nvpair name="SID"' $cib | sed -n "/value=\"${RSID}\"/s/.*id=\"\(.*\)-instance_attributes-.*>/\1/gp")"
```

Comment:
looks a bit fragile for me.

Reply:
@angelabriel  just stumbled across that. Just to read attributes or CIB values, maybe would make more sense to use cibadmin XPath query or crm_attribute. It would probably simplify the shell here.  Some example below:

``` cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/@type" ```

or 

```cf_status=$(crm_attribute -t crm_config -G -n concurrent-fencing --quiet)```